File: 2023/java/src/test/java/org/frieder/aoc/day3/TestDay3B.java
package org.frieder.aoc.day3;

import org.frieder.aoc.day3.b.Solution3B;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay3B {
    @Test
    public void testExample() throws IOException {
        assertEquals(467835, Solution3B.getResult("../resources/3/test_b.txt"));
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(78826761, Solution3B.getResult("../resources/3/input.txt"));
    }
}

File: 2023/php/test/TestDay3B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day3\b\Solution3B;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay3B extends TestCase
{
    public function testExample(): void
    {
        $this->assertSame(467835, Solution3B::getResult("../resources/3/test_b.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(78826761, Solution3B::getResult("../resources/3/input.txt", TestLogger::createTestLogger()));
    }

}